### PR TITLE
Add DPIAs  as document attachment rather  than a fixed (URL) attribute

### PR DESCRIPTION
**Pull request by barrydjenaba**
Fixing issue  #217

### CODE REVIEW

File: core/fixtures/document-types.json [hunk 1]
```diff
@@ -1,27 +1,33 @@
...
-  },
-  {
-    "name": "Subject Information Sheet Template"
-  }
+    {
+        "name": "Other"
+    },
+
+    {
+        "name": "DPIA"
```
Comment (pinarpink) [inline core/fixtures/document-types.json:7]: Dear @nene can we replace the short hand with the expanded description i.e.  "Data Protection Impact Asssessment"

File: core/models/document.py [hunk 2]
```diff
@@ -30,6 +30,7 @@ class Document(CoreModel):
                    ("consent_form", "Consent Form"),
                    ("subject_informationsheet", "Subject InformationSheet"),
                    ("project_proposal", "Project Proposal"),
+                   ('dpia', 'DPIA'),
```
Comment (pinarpink) [inline core/models/document.py:33]: Dear @nene can we replace the short hand  i.e. "DPIA" with the expanded description i.e.  "Data Protection Impact Asssessment"

File: core/models/project.py [hunk 3]
```diff
@@ -61,9 +61,7 @@ class AppMeta:
                                    null=True,
                                    help_text='Lay summary should provide a brief overview of project goals and approach. Lay summary may be displayed publicly if the project\'s data gets published in the data catalog')
 
-    dpia = models.TextField(verbose_name='DPIA Link',
```
Comment (pinarpink) [inline core/models/project.py:64]: While this field is removed from the model, the migration script does not contain any corresponding change to the database. Are you sure you ran makemigrations after removing this field?

File: core/fixtures/document-types.json (hunk 1 above)
Comment (barrydjenaba) [reply core/fixtures/document-types.json]: sure, already replaced

File: core/models/project.py (hunk 3 above)
Comment (barrydjenaba) [reply core/models/project.py:64]: Fixed it. thank you

**Review by pinarpink — APPROVED** [review]
@barrydjenaba this looks OK now. The only thing is PR #243 also has a migration script numbered 020. We might want to sort that out. Either we rebase this on develop after the merge of 243 or we merge and fix the migration script names on develop. @vildead what is your suggestion?

**Pull request merged**